Common/src/main/java/com/tabnineCommon/chat/commandHandlers/InitHandler.kt

```diff
@@ -21,11 +22,16 @@ data class InitPayload(
 
 class InitHandler(gson: Gson) : ChatMessageHandler<Unit, InitPayload>(gson) {
     override fun handle(payload: Unit?, project: Project): InitPayload {
-        val colorPalette = readColorPalette()
-        val isDarkTheme = EditorColorsManager.getInstance().isDarkEditor
-        val font = EditorColorsManager.getInstance().globalScheme.getFont(EditorFontType.PLAIN).size
+        var result: InitPayload? = null
```

Comment:
why not return a future hare and manage the flow async?

or at least i think there should be thread safe  wrapper around this